doc/source/serve/http-guide.md

```diff
@@ -1,13 +1,36 @@
-# Calling Deployments via HTTP
+# HTTP with Serve
 
 
+
+This section should help you understand how to:
+
+- configure HTTP server location
+- send HTTP request to serve deployment
+- use Ray Serve integrate with FastAPI
+- use customized HTTP Adapter
+
+
+## Configuring HTTP Server Locations
+
+By default, Ray Serve starts a single HTTP server on the head node of the Ray cluster.
+You can configure this behavior using the `http_options={"location": ...}` flag
+in {mod}`serve.start <ray.serve.start>`:
+
+- **HeadOnly**: start one HTTP server on the head node. Serve
+  assumes the head node is the node you executed serve.run
+  on. This is the default.
+- **EveryNode**: start one HTTP server per node.
+- **NoServer** or `None`: disable HTTP server.
+
+:::{note}
+To achieve high availability of Serve's HTTP proxies, use **EveryNode** option to point to the instance group of Ray cluster
```

Comment:
```suggestion
To achieve high availability of Serve's HTTP proxies, use the **EveryNode** option to point to the instance group of Ray cluster
```
What does instance group mean here? Maybe we should also mention using an external load balancer here, unless it'll be obvious to most readers